java-racingcar/src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("문자열을 쉼표로 분리하여 배열로 반환")
+    void splitString() {
+        String input = "1,2";
+        String[] result = input.split(",");
+        assertThat(result).contains("1", "2");
+        assertThat(result).containsExactly("1", "2");
```

Comment:
```suggestion
        assertThat(result).containsExactly("1", "2");
```
굳이 중복으로 확인할 필요가 있을까?
테스트 코드 또한 유지보수해야할 코드인 만큼 가능한 간단히 작성해 보는 것은 어떨까?